src/main/java/com/gbtec/gameoflife/basic/GameOfLifeCtrl.java

```diff
@@ -0,0 +1,59 @@
+package com.gbtec.gameoflife.basic;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+
+import com.gbtec.gameoflife.custom.GameOfLifeService;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@RestController
+public class GameOfLifeCtrl {
+
+    private final GameOfLifeService gameOfLifeService;
+
+    /**
+     * Init should trigger the initialization or reset of game of life and deliver the data to the view via websockets
```

Comment:
Use JavaDoc. Remove `should`.
```suggestion
     * <p> Triggers the initialization or resetting of game of life and delivers the data to the view via websockets
```

Reply:
still open